tools/featurecounts/README.rst

```diff
@@ -0,0 +1,55 @@
+FeatureCounts wrapper for Galaxy
```

Comment:
of deprecated tools in hun huidige repos laten staan tenzij we ze nog van plan zijn ze echt te gaan onderhouden